rewrites.js

```diff
@@ -309,10 +309,18 @@ exports.REWRITES = [
     source: '/how-it-works',
     destination: '/how-it-works',
   },
+  {
+    source: '/api/support',
+    destination: '/api/support',
+  },
```

Comment:
Can we remove this?

Reply:
@znarf This was the only way I could get it working. It did not work without this.